dist/index.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <script src="https://kit.fontawesome.com/a1316b0e7e.js" crossorigin="anonymous"></script>
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Nunito:wght@300&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <div id="headbar">
+        <div class="dropdown">
+            <button class="dropbtn"><i class="fas fa-bars"></i></button>
+            <div class="dropdown-content">
+              <p onclick="renderExpenses('fun')">Fun</p>
+              <p onclick="renderExpenses('food')">Food</p>
+              <p onclick="renderExpenses('rent')">Rent</p>
+              <p onclick="renderExpenses('bills')">Bills</p>
+              <p onclick="renderExpenses('misc')">Misc</p>
+              <p onclick="renderExpenses('all')">All</p>
+            </div>
+          </div>
```

Comment:
a better practice for this would be using the `label`, `select`, and `option` HTML tags